### ISSUE TITLE
keyboard.png not shown when simulator hosted on linux

### ISSUE THREAD

**Issue by pingu8007** [kenakamu/LINESimulator#3]
Image path in simulator.html not match to filename in folder:

* keyboard.png <-> keyboard.PNG

## How to reproduce:

1. run simulator on a `linux` box
2. open simulator and show keyboard

## Expected result:

Show keyboard

## Actual result:

Keyboard returned with 404

**Comment by kenakamu** [comment]
Thanks for the report. I don't have Linux bot at the moment but I will try to see where is the bug.

**Comment by kenakamu** [comment]
I could repro the issue even with Windows and it seems file extension became PNG rather than png. Can you try use the latest one?

**Comment by pingu8007** [comment]
Yes, it works. But the package comes from npm and it's not a good idea to ask user changing it manually.

I'm working on this and another issue related to richmenu, PR later.

**Comment by kenakamu** [comment]
Cool! thanks. The thing is I stop updating npm browser version and moving to native-app version now. How do you think?

**Comment by pingu8007** [comment]
Sounds good, it will be helpful for local developer if it can run without contacting to real LINE API (maybe blocked by firewall or whatever).
By far, however, I would prefer to browser version because I could leave them all inside a remote linux box and demo from anywhere that having browser. Our company having a ridiculous policy on both network accessing and program installing and I can't make it work without remote box.
